### ISSUE TITLE
DNS Brute is broken

### ISSUE THREAD

**Issue by iTosun** [saeeddhqan/Maryam#3]
Looks like the DNS Brute option concatenates the sub domain names wrong.
It goes wrong on line: https://github.com/saeeddhqan/Maryam/blob/master/modules/osint/dns_search.py#L94
Where the same variable is reused on line: https://github.com/saeeddhqan/Maryam/blob/master/modules/osint/dns_search.py#L144
The result of this bug is that it concatnates the subdomain in front of the url thus creating invalid url's
The following replacement of line 144 would be sufficient to fix this problem:
```
tmp_name = "https://{0}.{1}".format(i, host.split('://')[1])
```
Be warned the HTTPS is hardcoded thus in case of HTTP it would not work. (But still in the future HTTP would be obsolute because of enforced certificate errors)

**Comment by iTosun** [comment]
I've also noticed something weird happening on line 155-158, the results don't match with the previous output.

**Comment by saeeddhqan** [comment]
Thank for informing.
Fixed (DNS Brute is broken #3)
